specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1102,6 +1102,232 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/schemas": {
```

Comment:
Is schemas a tracked or a untracked resource. If tracked PATCH would be needed. But I am guessing this is untracked resource. Please confirm

Reply:
untracked resource

---
In reply to: [143310482](https://github.com/Azure/azure-rest-api-specs/pull/1801#discussion_r143310482) [](ancestors = 143310482)